modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -272,15 +274,15 @@ The Instance Metadata Service settings cannot be changed after your cluster is c
 ====
 endif::openshift-rosa[]
 +
-<5> If you have more than one set of account roles for your cluster version in your AWS account, an interactive list of options is provided.
-<6> Optional: Specify an unique identifier that is passed by {product-title} and the OpenShift installer when an account role is assumed. This option is only required for custom account roles that expect an external ID.
-<7> By default, the cluster-specific Operator role names are prefixed with the cluster name and a random 4-digit hash. You can optionally specify a custom prefix to replace `<cluster_name>-<hash>` in the role names. The prefix is applied when you create the cluster-specific Operator IAM roles. For information about the prefix, see _Defining an Operator IAM role prefix_.
+<6> If you have more than one set of account roles for your cluster version in your AWS account, an interactive list of options is provided.
+<7> Optional: Specify an unique identifier that is passed by {product-title} and the OpenShift installer when an account role is assumed. This option is only required for custom account roles that expect an external ID.
+<8> By default, the cluster-specific Operator role names are prefixed with the cluster name and a random 4-digit hash. You can optionally specify a custom prefix to replace `<cluster_name>-<hash>` in the role names. The prefix is applied when you create the cluster-specific Operator IAM roles. For information about the prefix, see _Defining an Operator IAM role prefix_.
```

Comment:
I was expecting "Defining an Operator IAM role prefix" to be in the same document/doc book, but it is not. Maybe at a link in an add. resources section?

Reply:
Cheers, I've corrected the title and added to the existing additional resources section